index.js

```diff
@@ -26,5 +26,12 @@ module.exports = function linearArray(n, offset = false) {
     throw new Error('expected a number greater than 0');
   }
 
-  return !offset ? [...Array(n).keys()] : Array.from(Array(n), (_, i) => i + 1);
+  const offsetValue = offset | 0;
```

Comment:
Can you remove this line, and use a default parameter value.

```
module.exports = function linearArray(n, offset = 0) {  ...  }
```

Currently the function expects a boolean as a second argument which enables only an offset of 1. But I think that this should be changes like in your PR. The offset should be a number so that the array can start from that specific point.